docs/fundamentals/database-collection.txt

```diff
@@ -0,0 +1,198 @@
+
+       'mongodb_alt' => [
+           'driver' => 'mongodb',
+           'dsn' => env('DB_URI', '<connection string>'),
+           'database' => 'plants',
+       ]
+
+   ], ...
+
+.. note::
+
+   The MongoDB PHP driver reuses the same connection when
+   you create two clients with the same connection string. There is no
+   overhead in using two connections for two distinct databases, so you
+   do not need to optimize your connections.
+
+If your application contains multiple
+database connections, you override the ``$connection``
+property when creating a ``Model`` class to apply the model to an
+alternate database.
+
+The following example shows how to override the ``$connection`` property
+on the ``Flower`` model class to use the ``mongodb_alt`` connection:
+
+.. code-block:: php
+
+   class Flower extends Model
+   {
+       protected $connection = 'mongodb_alt';
+   }
+
+.. _laravel-access-coll:
+
+Access a Collection
+-------------------
+
+When you create a ``Model`` class in {+odm-short+}, it is automatically
+stored in a collection with a name that is the plural of the model class
+name. For example, if you create a model class called ``Flower``,
+Laravel applies the model to the ``flowers`` collection in the database.
+
+.. tip::
+
+   You might be storing data in a collection with a different name than the
+   model, or your collection might contain more than one type of data. To
+   learn how to set a collection name when defining a model class, see the
+   :ref:`laravel-model-customize-collection-name` section of the Eloquent
+   Model Class guide.
+
+For many operations, such as CRUD operations and relationships, we
+recommend that you use the Eloquent ORM to access a collection
+implicitly. The following example specifies a find operation by using
+the ``Flower`` class, so Laravel retrieves results from the ``flowers``
+collection:
+
+.. code-block:: php
+
+   Flower::where('name', 'Water Lily')->get()
+
+For other collection operations, you can access a collection by using
+a facade to access the query builder class ``DB`` and calling the
+``collection()`` method. The following example shows the same query as
+in the preceding example constructed by using the ``DB::collection()``
+method:
+
+.. code-block:: php
+
+   DB::connection('mongodb')
+       ->collection('flowers')
+       ->where('name', 'Water Lily')
+       ->get()
+
+.. tip::
+   
+   In MongoDB, each document in a collection has a field called ``_id`` which has a value
+   that is a unique identifier. To learn more about customizing this field
+   when you create a model to access a collection, see the :ref:`laravel-model-customize-primary-key` field of the Eloquent
+   Model Class guide.
+   
+List Collections
+----------------
+
+To see the names of the collections in a database, call the 
+``listCollections()`` method.
+
+The following example acceses a database connection, then
+calls the ``listCollections()`` method and outputs the collection names:
```

Comment:
[`Database::listCollections()`](https://www.mongodb.com/docs/php-library/current/reference/method/MongoDBDatabase-listCollections/) is going to yield a list of documents _describing_ the collection. The name is included, but it's not the only field.

In contrast, [`listCollectionNames()`](https://www.mongodb.com/docs/php-library/current/reference/method/MongoDBDatabase-listCollectionNames/) returns a list of strings.

Here, it would be more accurate to say that you're outputting information about each collection in the database instead of just the "names".